Alloy/builtins/animation.js

```diff
@@ -20,8 +20,8 @@ exports.HORIZONTAL = 'horizontal';
  */
 exports.VERTICAL = 'vertical';
 
-const create3DMatrix = Ti.version >= '8.0.0' ? Ti.UI.createMatrix3D : Ti.UI.create3DMatrix;
-const create2DMatrix = Ti.version >= '8.0.0' ? Ti.UI.createMatrix2D : Ti.UI.create2DMatrix;
+const create3DMatrix = Ti.UI.createMatrix3D ? Ti.UI.createMatrix3D : Ti.UI.create3DMatrix;
+const create2DMatrix = Ti.UI.createMatrix3D ? Ti.UI.createMatrix2D : Ti.UI.create2DMatrix;
```

Comment:
There's a typo in the 2nd line. You should be checking for the 2D method's existence, not the 3D method.

Reply:
Doh, fixed!